sdk/communication/azure-communication-email/tests/conftest.py

```diff
@@ -25,12 +25,12 @@
 # --------------------------------------------------------------------------
 import pytest
 import os
-from devtools_testutils import test_proxy, add_general_regex_sanitizer, add_header_regex_sanitizer, set_default_settings, add_uri_regex_sanitizer
+from devtools_testutils import test_proxy, add_general_regex_sanitizer, add_header_regex_sanitizer, set_default_function_settings, add_uri_regex_sanitizer
 from azure.communication.email._shared.utils import parse_connection_str
 
 @pytest.fixture(scope="session", autouse=True)
 def add_sanitizers(test_proxy):
-    set_default_settings()
+    set_default_function_settings()
```

Comment:
what is this change about?

Reply:
the email team imported an undefined function from devtools_testutils and since then the pipeline test run was failing for communication modalities, so I fixed it